src/main/java/edu/wpi/first/gradlerio/OneDriveException.java

```diff
@@ -6,5 +6,6 @@ public class OneDriveException extends java.lang.RuntimeException {
   public OneDriveException(Project project) {
     super(String.format("Error cannot create project inside OneDrive. Project Directory = %S", project.getRootDir().toString()));
     System.out.println(String.format("Error cannot create project inside OneDrive. Project Directory = %S", project.getRootDir().toString()));
+    System.out.println("To fix this error move your project out of the OneDrive folder or select a different directory in the project creator.");
```

Comment:
I'm not sure the part about the project creator is helpful here. By the time this error happens, the project would have already been created, so the solution is to move. To address project create, the error message that should be updated is here: https://github.com/wpilibsuite/vscode-wpilib/blob/ccc008aef0ad04ddcd2595edb4d982250504df30/vscode-wpilib/src/webviews/pages/sharedpages.ts#L36